src/sentry/sentry_metrics/indexer/models.py

```diff
@@ -32,3 +33,22 @@ def get_next_values(cls, num: int) -> Any:
             "SELECT nextval('sentry_metricskeyindexer_id_seq') from generate_series(1,%s)", [num]
         )
         return connection.fetchall()
+
+
+class StringIndexer(Model):  # type: ignore
+    __include_in_export__ = False
+
+    string = models.CharField(max_length=200)
+    organization_id = BoundedBigIntegerField()
+    project_id = BoundedBigIntegerField()
+    date_added = models.DateTimeField(default=timezone.now)
+    last_seen = models.DateTimeField(default=timezone.now)
```

Comment:
Will we be using buffers to update this?

Reply:
@wedamija you mean like how we do here?https://github.com/getsentry/sentry/blob/7baf8df7d909ffde827d42927e0e69c17ff04bc3/src/sentry/models/grouprelease.py#L70-L77

That seems reasonable to do. I also need to add an index to this since it will be used for TTL deletions

Reply:
Yeah, it's important that we do this to help reduce load on Postgres. I assume that there will be a reasonably high rate of writes here. Is it basically updated whenever a release is seen?

Reply:
Not sure it will be exactly that buffer implementation (likely, though), but certainly we will do debouncing of the updates. The advantage here is that we do not have to track updates for each message the consumer sees. We just need to update the last seen every few hours as this is just meant for TTL.

Reply:
Since the `last_seen` only needs to be updated every few hours we might want to use a different solution, since buffers flush every 90 seconds or so. I don't think it's possible to set custom flush times for different buffers at the moment, that could be one option.

Reply:
We have a long cache in the indexer so that it loads data from postgres only every 4 hours. We can queue the updates only when we have a cache miss.